src/main/kotlin/no/nav/familie/ef/sak/journalføring/JournalføringService.kt

```diff
@@ -72,12 +74,37 @@ class JournalføringService(private val journalpostClient: JournalpostClient,
 
     @Transactional
     fun fullførJournalpost(journalføringRequest: JournalføringRequest, journalpostId: String): Long {
+        return if (journalføringRequest.skalJournalførePåEksisterendeBehandling()) {
+            journalførSøknadTilEksisterendeBehandling(journalføringRequest, journalpostId)
+        } else {
+            journalførSøknadTilNyBehandling(journalføringRequest, journalpostId)
+        }
+    }
+
+    private fun journalførSøknadTilEksisterendeBehandling(journalføringRequest: JournalføringRequest,
+                                                          journalpostId: String): Long {
         val saksbehandler = SikkerhetContext.hentSaksbehandler(true)
         val behandling: Behandling = hentBehandling(journalføringRequest)
         val journalpost = hentJournalpost(journalpostId)
         val fagsak = fagsakService.hentFagsak(journalføringRequest.fagsakId)
+        knyttJournalpostTilBehandling(journalpost, behandling)
+        oppdaterJournalpost(journalpost, journalføringRequest.dokumentTitler, fagsak.eksternId.id, saksbehandler)
+        ferdigstillJournalføring(journalpostId, journalføringRequest.journalførendeEnhet, saksbehandler)
+        ferdigstillJournalføringsoppgave(journalføringRequest)
+        return journalføringRequest.oppgaveId.toLong()
+    }
 
-        iverksettService.startBehandling(fagsak)
+    private fun journalførSøknadTilNyBehandling(journalføringRequest: JournalføringRequest,
+                                                journalpostId: String): Long {
+        val saksbehandler = SikkerhetContext.hentSaksbehandler(true)
+        val behandling = opprettBehandlingMedBehandlingstype(journalføringRequest.behandling.behandlingstype,
+                                                             journalføringRequest.fagsakId)
+        val journalpost = hentJournalpost(journalpostId)
+        val fagsak = fagsakService.hentFagsak(journalføringRequest.fagsakId)
+
+        if (journalføringRequest.behandling.behandlingstype == BehandlingType.FØRSTEGANGSBEHANDLING) {
+            iverksettService.startBehandling(fagsak)
```

Comment:
Dette blir kokos - vi tillater kun én `startBehandling` per fagsak (fagsakId er task-payload). 
Bør `startBehandling` sette et flagg på fagsaken (evt egen tabell) som sier at vi har trigget denne - og følgelig sjekker vi det flagget istedenfor førstegangsbehandling?

Kan jo ha to førstegangsbehandlinger (den første avslås / henlegges) - og da vil denne smelle 😱 💥 😱